src/components/panel/panel.js

```diff
@@ -311,17 +314,17 @@ function MdPanelService($rootElement, $injector) {
 /**
  * Creates a panel with the specified options.
  * @param {!Object=} opt_config Configuration object for the panel.
- * @returns {!$MdPanelReference}
+ * @returns {!$mdPanelReference}
  */
 MdPanelService.prototype.create = function(opt_config) {
   var configSettings = opt_config || {};
 
   angular.extend(this._config, configSettings);
 
-  var instanceID     = $injector.get('$mdUtil').nextUid();
-  var instanceConfig = angular.extend({ }, this._config, {id : instanceID } );
+  var instanceId = 'panel_' + this._$injector.get('$mdUtil').nextUid();
+  var instanceConfig = angular.extend({ }, this._config, {id : instanceId } );
```

Comment:
(nit) Why not: 

var instanceConfig = angular.extend({id : instanceId }, this._config);